packages/core/src/Modules/Cashier/Components/Form/account-transfer-form.jsx

```diff
@@ -319,7 +319,6 @@ const AccountTransferForm = ({
                                             {...field}
                                             onChange={e => {
                                                 setErrorMessage('');
-                                                setFieldTouched('amount', true);
```

Comment:
This line is to provide real time validation to display error hint, you may find a workaround. you can refer to [this card](https://trello.com/c/nEp4WlCS/3047-matin-error-message-only-display-after-out-of-focus)